src/nupic/algorithms/KNNClassifier.py

```diff
@@ -356,7 +356,9 @@ def learn(self, inputPattern, inputCategory, partitionId=None, isSparse=0,
     @param inputCategory (int) The category to be associated to the training
         pattern
 
-    @param partitionId (int) UNKNOWN
+    @param partitionId (int) The k-fold cross validation partition for this
```

Comment:
I'm pretty sure that is the intended use including the using standard k-folds terminology of "partition." One may be able to find a different way of using the functionality, but I expect that such alternate uses would be much less common.